docs/MarketingCloud-batchsource.md

```diff
@@ -92,9 +92,6 @@ For example, `https://instance.auth.marketingcloudapis.com/`
 **SOAP API Endpoint**: The SOAP Endpoint URL associated for the Server-to-Server API integration. For example, 
 `https://instance.soap.marketingcloudapis.com/Service.asmx`
 
-**REST API Base URI**: The REST API Base URL associated for the Server-to-Server API integration. For example, 
```

Comment:
why was this removed?

Reply:
It was not being used in sink. That's why I had removed it.